charts/fybrik/values.yaml

```diff
@@ -70,6 +70,14 @@ global:
     capabilities:
       drop:
       - ALL
+  tls:
```

Comment:
I don't think TLS setting should be global, probably we need to configure it per server.

Reply:
OK I will change tls and mtls per server/client connection.